### PR TITLE
chore: update changelog for next release

### PR DESCRIPTION
**Pull request by Doug-Reed**
updates changelog for 9.0.0

----

PR considerations checklist:

- [X] Updates `CHANGELOG.md` to reflect this PR's change.
- [X] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
 `optionsTemplateURL` will display the template in the side navigation menu.
 This may require minor layout/appearance adjustments in the template. (#684)
 
++ To upgrade:
```
Comment (ChristianMurphy) [inline CHANGELOG.md]: :art: `23:3  warning  Incorrect list-item indent: add 2 spaces  list-item-indent  remark-lint`

I suspect this is being confused by lines 24-26, the parser isn't seeing enough spaces to determine these are sub-list items.

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
...
+  ++ If using the "app-title" attribute on frame-page directive, that title will be used as the document title
+  ++ If you want a heading on pages that formerly had the app-header, you'll need to use the "page-title" attribute on frame-page directive (or add your own <h1> tag to their pages wherever/however you want)
```
Comment (apetro) [inline CHANGELOG.md]: I think this language we so often reach for about what "you want" is problematic. It's not about what you or I want. It's about building applications on behalf of the institution that serve users well. 

Do we really want to be encouraging app developers to sprinkle in ad-hoc `<h1>` tags when they could be feeding a `page-title` to `frame-page` and thereby making these titles available semantically in a way that might migrate forward in future releases?

So what about something more like

```
++ Pages formerly using `app-header` can use the `page-title` attribute on `frame-page` to achieve a similar within-page `h1` heading as before. 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +20,11 @@ unaffected. (#684)
...
+  ++ If you were previously using APP_OPTIONS.optionsTemplateURL to set up app-header options, those options will now appear in the side navigation and may likely need a bit of tweaking if you had a rigid CSS layout.
```
Comment (apetro) [inline CHANGELOG.md]: Likewise. "you" weren't using `APP_OPTIONS.optionsTemplateURL`. Maybe you have a role in developing or  maintaining an app that uses this.

**Comment by vertein** [comment]
Looks like the CI checks were failing and some were running when this was merged?

**Comment by apetro** [comment]
Good catch on the `upgrading.md`, @vertein . I'd forgotten about that.

**Pull request merged**